transform/operations.go

```diff
@@ -239,11 +239,32 @@ func (t *timeParse) transform(raw interface{}) (interface{}, error) {
 	if !ok {
 		return nil, errors.New("timeParse only supports strings")
 	}
-	parsedTime, err := time.Parse(t.Args["format"], in)
+	parsedTime, err := time.Parse(t.args["format"], in)
 	if err != nil {
 		return nil, fmt.Errorf("time could not be parsed using supplied format")
 	}
-	return parsedTime.Format(t.Args["layout"]), nil
+	return parsedTime.Format(t.args["layout"]), nil
+}
+
+type currentTime struct {
+	args map[string]string
+}
+
+func (c *currentTime) init(args map[string]string) error {
+	if err := requiredArgs([]string{"format"}, args); err != nil {
+		return err
+	}
+	c.args = args
+	return nil
+}
+
+func (c *currentTime) transform(raw interface{}) (interface{}, error) {
+	timeFmt := c.args["format"]
+	switch c.args["format"] {
+	case "RFC3339":
+		timeFmt = time.RFC3339
```

Comment:
💯

Reply:
I'm confused... by this. Format takes a string. so why all this extra stuff when what you actually want is to type assert the format arg to a string and then use it?

Reply:
bc passing in "RFC3339" as an argument cant be used directly as its meaningless. the time.Format() just takes it as a string instead of grabbing the const from the time package. So we want means to pass in the typical go time formatting date as well as shorthand versions on a case by case basis

Reply:
right... but I woudl pass in time.RFC3339 as the argument. not RFC3339

Reply:
It's just a convenience method for clarity rather than having to actually find the constant value for `time.RFC3339` and reading over the transforms and understanding that's what it is. 

https://golang.org/pkg/time/
```golang
        RFC3339     = "2006-01-02T15:04:05Z07:00"
```

Reply:
like. RFC3339 is the name of a format not an actual format.

Reply:
since we are doing it via a config in json we can't do it like that. this is correct just sad.